### PR TITLE
add Duration::ofMilliseconds constructor

### PR DESCRIPTION
**Pull request by Solodkiy**

### CODE REVIEW

File: src/Duration.php [hunk 1]
```diff
@@ -163,6 +163,21 @@ public static function ofSeconds(int $seconds, int $nanoAdjustment = 0) : Durati
...
+    /**
+     * Returns a Duration from a number of milliseconds.
+     *
+     * @param int $milliseconds
+     *
+     * @return Duration
+     */
+    public static function ofMilliseconds(int $milliseconds) : Duration
+    {
+        $seconds = intval($milliseconds / LocalTime::MILLIS_PER_SECOND);
```
Comment (BenMorel) [inline src/Duration.php]: Please use `intdiv()`, otherwise the result is a float casted to int.

File: src/Duration.php [hunk 2]
```diff
@@ -163,6 +163,21 @@ public static function ofSeconds(int $seconds, int $nanoAdjustment = 0) : Durati
...
+        $nanos = ($milliseconds % LocalTime::MILLIS_PER_SECOND) * intval(LocalTime::NANOS_PER_SECOND / LocalTime::MILLIS_PER_SECOND);
```
Comment (BenMorel) [inline src/Duration.php]: No need for `intval()` here, the division will always yield an `int`.

File: tests/DurationTest.php [hunk 3]
```diff
@@ -48,6 +48,27 @@ public function providerOfSeconds() : array
         ];
     }
 
+    /**
+     * @param int $miliseconds
+     * @dataProvider providerOfMilliseconds
+     */
+    public function testOfMilliseconds(int $miliseconds)
```
Comment (BenMorel) [inline tests/DurationTest.php]: Typo: `$milliseconds`

**Comment by BenMorel** [comment]
Looks good to me, once the suggestions are implemented! 👍

**Comment by Solodkiy** [comment]
@BenMorel Done.

**Comment by BenMorel** [comment]
@Solodkiy Thanks! Merged.

**Pull request merged**